src/buildHTML.js

```diff
@@ -1338,15 +1358,11 @@ groupTypes.kern = function(group, options) {
     // Make an empty span for the rule
     const rule = makeSpan(["mord", "rule"], [], options);
 
-    let dimension = 0;
     if (group.value.dimension) {
-        dimension = calculateSize(group.value.dimension, options);
+        const dimension = calculateSize(group.value.dimension, options);
+        rule.style.marginLeft = dimension + "em";
     }
 
-    dimension /= options.sizeMultiplier;
-
-    rule.style.marginLeft = dimension + "em";
```

Comment:
Not sure why were doing this out side of the `if` statement.  Thanks for cleaning this up.